src/darker/tests/test_main.py

```diff
@@ -39,15 +44,14 @@ def test_isort_option_with_isort(run_isort):
     assert "Please run" not in run_isort.caplog.text
 
 
-def test_isort_option_with_isort_calls_sortimports(run_isort):
+@pytest.mark.parametrize(
+    "run_isort,isort_args",
```

Comment:
I've preferred to always delimit the `parametrize` argument list with spaces so it nicely matches the function signature visually.

Cool trick with `request.param`! I had forgotten about the ability to pass parameters to fixtures. Thanks!

```suggestion
    "run_isort, isort_args",
```